src/host/configuration.h

```diff
@@ -13,14 +13,14 @@ namespace host
 {
   enum class EnclaveType
   {
-    RELEASE,
-    DEBUG,
-    VIRTUAL
+    SGX_RELEASE,
+    SGX_DEBUG,
+    VIRTUAL,
   };
   DECLARE_JSON_ENUM(
     EnclaveType,
-    {{EnclaveType::RELEASE, "release"},
-     {EnclaveType::DEBUG, "debug"},
+    {{EnclaveType::SGX_RELEASE, "release"},
+     {EnclaveType::SGX_DEBUG, "debug"},
```

Comment:
This PR is also baby steps towards clean multi-platform support. I wonder if we should be explicit now and rename `"release"` to `"sgx_release"`? But I think it's an unnecessary breaking change, and may prove unnecessary, depending what those platforms look like.

Reply:
Not all platforms will use an executable + .so, so I don't think we should worry about this too much for now. It's also easier and more compatible to create "new_platform_release" than rename existing types.